blue-green-deploy/README.md

```diff
@@ -0,0 +1,86 @@
+# Blue Green Deploy on Kubernetes
+
+THe following task can help you to deploy an application using the Blue-Green deployment strategy. This task gives you the flexibility to deploy a version of the application from the blue zone to the green zone without disturbing the previous version in the blue zone. It works by making the service point to the newer version of the application deployed. 
+
+- If you are deploying the first version of the application then it will go into the blue zone and for that you need to provide deployment and service manifests via the `workspaces` and in the `params` the service name and version. Sample Kubernetes manifests for version v1 can be found [here](./v1-deploy).
+- For further deployments the deployments will get deployed in the other zone to which the current service is pointing to. For example if we have a deployment running in the blue zone then we will deploy the next deployment in the green zone and make the service point to the green zone or vice versa.
```

Comment:
`For further deployments the deployments will get deployed in the other zone to which the current service is pointing to.`
is that current service points to new deployment ?